lib/broccoli/ember-app.js

```diff
@@ -286,7 +286,9 @@ EmberApp.prototype._initVendorFiles = function() {
         }
       }
     ]
-  }, this.options.vendorFiles), isNull);
+  }, this.options.vendorFiles), function(value) {
```

Comment:
I am not seeing how this is any different that what we already have since `_.isNull` is actually [the same code](https://github.com/lodash/lodash/blob/4.3.0/lodash.js#L10063):

```js
    function isNull(value) {
      return value === null;
    }
```

I'm also confused how the merging of `vendorFiles` is related to `outputPaths`.